data/src/main/java/io/gnosis/data/repositories/ChainInfoRepository.kt

```diff
@@ -0,0 +1,28 @@
+package io.gnosis.data.repositories
+
+import io.gnosis.data.backend.GatewayApi
+import io.gnosis.data.db.daos.ChainDao
+import io.gnosis.data.models.Chain
+import io.gnosis.data.models.ChainInfo
+
+class ChainInfoRepository(
```

Comment:
how about putting chain related stuff into SafeRepository?

Reply:
I put it there at first. But there might be parts of the app that want to get network info without using a safe.